home/home2.1/AdvancedJava.java

```diff
@@ -0,0 +1,90 @@
+import java.util.HashMap;
+import java.util.Map;
+import java.io.BufferedReader;
+import java.io.FileWriter;
+import java.io.FileReader;
+import java.io.BufferedWriter;
+class Constants {
+    public static final String delete = "delete";
+    public static final String bread = "bread";
+}
+class Database {
```

Comment:
You will be able to adjust java version of program with c++ fixed implementation  in mind